xarray/tests/test_backends.py

```diff
@@ -552,6 +552,19 @@ def test_ondisk_after_print(self):
             repr(on_disk)
             assert not on_disk['var1']._in_memory
 
+    def test_setncattr_string(self):
+        """
+        Test to ensure setncattr_string is called on lists of strings
+        with NetCDF4
+        """
```

Comment:
Nit: we generally avoid writing docstrings for test method unless they add substantial information beyond what's already obvious from the class/method name.